doc/radosgw/dynamicresharding.rst

```diff
@@ -0,0 +1,72 @@
+===================================
+RGW Dynamic Bucket Index Resharding
+===================================
+
+.. versionadded:: Luminous
+
+ A large bucket index can lead to performance problems. In order to address this problem we introduced bucket index sharding.
+ Until Luminous changing the number of bucket shards (resharding) required to offline the bucket, from Luminous we support
+ online bucket resharding.
+
+Each bucket index shard can handle efficiently up to a threshold of entries, if this threshold is exceeded the system
+will encounters performance issues.
+The dynamic resharding feature detects this situation and increase automatically the number of shards used by the bucket index,
+resulting in the reduction of the number of entries in each bucket index shard. This process is transparent to the user.
+
+The detection process consists of two parts:
+The first is executed during addition of new objects to the bucket and the second is  a background process that periodically scans all the
+buckets to handler older buckets in the system that are not being updated.
```

Comment:
s/to handler .../This is needed in order to deal with existing buckets that are not being updated.